Ecosystem Partners/Marketplace Guides/getting-started-with-datto-alto-blueprint.md

```diff
@@ -109,18 +109,14 @@ can start protecting data and virtualizing systems.
 
 The costs listed above in the above steps are for the infrastructure only.
 
-After deploying this Blueprint, you may secure entitlements to the technology by emailing your datto
-Account Executive or if you don’t have one, Brian Buckman at bbuckman@datto.com.
+After deploy this Blueprint, you may secure entitlements to the technology by contacting your authorized centurylink provider and 
+if you don't have one contact Brian Buckman at bbuckman@datto.com.
```

Comment:
separate this into a new sentence.  `...provider.  If you don't have one, contact...`